src/plots/wheelPlot.ts

```diff
@@ -0,0 +1,222 @@
+      attrToProjector["d"] = (datum: any, index: number, ds: Dataset) => {
+        let startAngle = startAngleAccessor(datum, index, ds) % (Math.PI * 2);
+        let endAngle = endAngleAccessor(datum, index, ds) % (Math.PI * 2);
+        while (endAngle < startAngle) { endAngle += Math.PI * 2; }
+        return d3.svg.arc().innerRadius(innerRadiusAccessor(datum, index, ds))
+                           .outerRadius(outerRadiusAccessor(datum, index, ds))
+                           .startAngle(startAngle)
+                           .endAngle(endAngle)(datum, index);
+      };
+      return attrToProjector;
+    }
+
+    /**
+     * Gets the AccessorScaleBinding for the start angle.
+     */
+    public startAngle<T>(): AccessorScaleBinding<T, number>;
+    /**
+     * Sets the start angle to a constant number or the result of an Accessor<number>.
+     *
+     * @param {number|Accessor<number>} startAngle
+     * @returns {Wheel} The calling Wheel Plot.
+     */
+    public startAngle(startAngle: number | Accessor<number>): Plots.Wheel<R, T>;
+    /**
+     * Sets the start angle to a scaled constant value or scaled result of an Accessor.
+     * The provided Scale will account for the values when autoDomain()-ing.
+     *
+     * @param {S|Accessor<number>} startAngle
+     * @param {Scale<T, number>} scale
+     * @returns {Wheel} The calling Wheel Plot.
+     */
+    public startAngle<T>(startAngle: T | Accessor<T>, scale: Scale<T, number>): Plots.Wheel<R, T>;
+    public startAngle<T>(startAngle?: number | Accessor<number> | T | Accessor<T>, scale?: Scale<T, number>): any {
+      if (startAngle == null) {
+        return this._propertyBindings.get(Wheel._START_ANGLE_KEY);
+      }
+
+      if (scale != null) {
+        scale.range([0, Math.PI * 2]);
+      }
+
+      let endAngleBinding = this.endAngle();
+      let endAngleAccessor = endAngleBinding && endAngleBinding.accessor;
+      if (endAngleAccessor != null) {
+        this._bindProperty(Wheel._END_ANGLE_KEY, endAngleAccessor, scale);
+      }
+
+      this._bindProperty(Wheel._START_ANGLE_KEY, startAngle, scale);
+      this.render();
+      return this;
+    }
+
+    /**
+     * Gets the AccessorScaleBinding for the end angle.
+     */
+    public endAngle<T>(): AccessorScaleBinding<T, number>;
+    /**
+     * Sets the end angle to a constant number or the result of an Accessor<number>.
+     *
+     * @param {number|Accessor<number>} endAngle
+     * @returns {Wheel} The calling Wheel Plot.
+     */
+    public endAngle(endAngle: number | Accessor<number>): Plots.Wheel<R, T>;
+    public endAngle<T>(endAngle?: number | Accessor<number> | T | Accessor<T>): any {
+      if (endAngle == null) {
+        return this._propertyBindings.get(Wheel._END_ANGLE_KEY);
+      }
+
+      let startAngleBinding = this.startAngle();
+      let angleScale = startAngleBinding && startAngleBinding.scale;
+      this._bindProperty(Wheel._END_ANGLE_KEY, endAngle, angleScale);
```

Comment:
I notice that this logic isn't symmetric with the logic in `startAngle()`, which causes some odd behavior. For example, this works:
```Typescript
plot.endAngle(function(d){ return d.t2; });
plot.startAngle(function(d){ return d.t1; }, tScale);
```
but this doesn't:
```Typescript
plot.startAngle(function(d){ return d.t1; });
plot.endAngle(function(d){ return d.t2; }, tScale);
```